src/lib/standalone/ubus.ts

```diff
@@ -4,20 +4,55 @@
 import axios from 'axios'
 import { getStandaloneApiEndpoint } from '../config'
 import { useLoginStore } from '@/stores/standalone/standaloneLogin'
+import { MessageBag } from '../validation'
+
+type ValidationError = {
+  message: string
+  parameter: string
+  value: string
+}
+
+export class NeValidationError extends Error {
```

Comment:
I'd rather change this in:

```suggestion
type ValidationErrorResponse = {
  message: string
  parameter: string
  value: string
}

export class ValidationError extends Error {
```

just to keep things without explicit branding since we don't actually need it.
What do you think?

Reply:
I put the prefix thinking that we might move it to the library someday, but for now let's follow your suggestion

Reply:
Don't think it will, many systems think differently on how to manage Validation Errors, better to keep business logic outside of a all-rounded library for components